tests/dragonfly/cluster_test.py

```diff
@@ -1294,10 +1294,11 @@ async def test_network_disconnect_during_migration(df_factory, df_seeder_factory
 
 
 @pytest.mark.parametrize(
-    "node_count, segments, keys",
+    "node_count, segments, keys, huge_values",
     [
-        pytest.param(3, 16, 20_000),
-        pytest.param(5, 20, 30_000, marks=[pytest.mark.slow, pytest.mark.opt_only]),
+        pytest.param(3, 16, 20_000, 10),
+        pytest.param(3, 16, 20_000, 1_000_000),
```

Comment:
do we have any containers in test with values of this size 1_000_000?

Reply:
No, that's why I used `10` above (to force splitting)

Reply:
so using 1_000_000 is to check only the old flow of only restore commands?
lets add a comment here 
and we do actually want to check both flows in one test I think it is better coverage 
we should edit seeder code to make different container I asked Kostas to create an issue for this
@kostasrim please add in the issue description that once this change is introduced we can all use it in this test

Reply:
done https://github.com/dragonflydb/dragonfly/issues/4157

Reply:
> so using 1_000_000 is to check only the old flow of only restore commands?

Yes, and I added a comment & TODO